explicit_coupling/README.md

```diff
@@ -0,0 +1,3 @@
+Now that LS-DYNA is installed the coupling can begin. 
+Here the LS-DYNA solver is coupled with a c++ dummy. 
+Paste the code from the `.f` file in the appropriate places in the dyn.21 file and run LS-DYNA with cube_userload.k as the input file. Afterwards the c++ dummy should be able to connect to LS-DYNA.
```

Comment:
I dont understand the relevance of these lines, are they parsed somewhere during runtime? If not then lets remove them as they are not adding much information